wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,41 +1,62 @@
+| ![](/wiki/shared/judgement/osu!/hit300k.png "300 Katu") | 300 (喝) Katu or Katsu | A variant of the 300 result. Originates from Ouendan, called Beat! in EBA. Granted upon hitting a 300 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 300. |
+| ![](/wiki/shared/judgement/osu!/hit100.png "100") | 100 | The second best achievable result for a hit object. Usually denoted with a green colour. |
+| ![](/wiki/shared/judgement/osu!/hit100k.png "100 Katu") | 100 (喝) Katu or Katsu | A variant of the 100 result. Originates from Ouendan, called Beat! in EBA. Granted upon hitting a 100 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 100. |
+| ![](/wiki/shared/judgement/osu!/hit50.png "50") | 50 | The third best achievable result for a hit object, the last one before a miss. Usually denoted with an orange colour. Scoring a 50 will prevent the appearance of a Katu or a Geki at the combo's end. |
+| ![](/wiki/shared/judgement/osu!/hit0.png "Miss") | Miss | The worst possible result for a hit object. Scoring a Miss will reset the current combo to 0 and will prevent the appearance of a Katu or a Geki at the combo's end. |
 
 ## Judgement mechanics
 
-**Hit circles:**
+### Hit circles
+
+- A hit circle is judged as a 300, 100, or 50 depending on how accurate the player's timing is when clicking on it.
+  The time intervals used to determine the concrete result for a hit circle are called *hit windows*, and are influenced by the beatmap's [overall difficulty](/wiki/Beatmap/Overall_difficulty#osu!).
+- A hit circle is judged as missed when:
+  - The circle is hit before the start of the 50 hit window and does not trigger [notelock](/wiki/Gameplay/Judgement/Notelock).
+  - The circle is hit after the end of the 50 hit window, or not hit at all.
+
+### Sliders
+
+Sliders consist of parts such as the slider head, slider tail, slider ticks, and slider repeats.
+The slider as a whole is judged based on how many parts of the slider the player has hit.
+
+- A 300 is given if the player successfully hits all slider parts.
+- A 100 is given if the player hits at least half of them.
+- A 50 is given if the player manages to hit at least one of the parts.
+- A miss is given if the player does not successfully hit any of the slider parts.
+- If [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) is active, the accuracy of hitting the slider head is additionally accounted for when judging the entire slider:
+  - Getting a 300 for a slider requires getting a 300 judgement for the slider head.
+  - Getting a 100 for a slider requires getting a 100 judgement or better for the slider head.
```

Comment:
perhaps this may be reworded into something less repetitive, so that the whole list is less `if x then y`. example:

```md
something something judgement is based on a no. of successfully cleared elements:

- 300: all parts
- 100: at least half of the parts
- 50: at least one part
- miss: no parts
```

same goes for the spinner-related section

Reply:
See how you find the latest changes. Hopefully better in this respect.

Reply:
that's better. the format likely couldn't be improved much as it's structure-dependent, so I'll take that